Unity-Technologies/ScriptableRenderPipeline
# SG Fix: Active Master Node Undo Fix + 2 Other Undo Fixes

**Summary:**
Fixes bugs related to lack of registering undos: namely 1 prio 3 bug and 3 other Undo related bugs (first two are basically the same bug):
https://fogbugz.unity3d.com/f/cases/1184932/
https://fogbugz.unity3d.com/f/cases/1215447/
https://fogbugz.unity3d.com/f/cases/1215448/
https://fogbugz.unity3d.com/f/cases/1213629/

---
**Manually Tested:**

* That none of the bug reproduces anymore
* For the default-value-sub-node, all sorts of changes to confirm that the title asterisk only appears when necessary, such as something that results in the field returning to it was before type (0, safeaef => 0)

---
**Technical Risk:** 1/4
**Halo Effect:** 1/4

---
**Yamato:**

https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/sg%252Ffix-unrecorded-undos
